custom/enikshay/management/commands/get_vouchers_for_bets.py

```diff
@@ -0,0 +1,47 @@
+import csv
+import json
+
+from django.core.management.base import BaseCommand
+
+from corehq.util.log import with_progress_bar
+from corehq.motech.repeaters.dbaccessors import iter_repeat_records_by_domain, get_repeat_record_count
+
+
+class Command(BaseCommand):
+    help = """
+    Output a CSV file of voucher cases that were sent to BETS in repeater <repeater_id>.
+
+    This should be run twice, once for Chemist vouchers, and once for Lab vouchers.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('repeater_id')
+        parser.add_argument('filename')
+
+    def handle(self, domain, repeater_id, filename, **options):
+        records = iter_repeat_records_by_domain(domain, repeater_id=repeater_id)
+        record_count = get_repeat_record_count(domain, repeater_id=repeater_id)
+
+        row_names = [
+            'EventID',
+            'EventOccurDate',
+            'BeneficiaryUUID',
+            'BeneficiaryType',
+            'Location',
+            'DTOLocation',
+            'VoucherID',
+            'Amount',
+            'succeeded',    # Some records did succeed when we sent them.
+                            # Include this so they don't re-pay people.
+        ]
```

Comment:
@Do we have any date information that's precise enough to be meaningful?  (`RepeatRecord.attempts.datetime` is a mediocre proxy)  I think BETS may want access to that if we have it.